BuildForSDG/Team-037-Product-backend
# ft: user can login with google and create the user data to the database

#### What does this PR do?
ft: user can login with google and create the user data to the database#### Description of Task to be completed?
list the things you did to complete this project
- i use https://console.developers.google.com/ to get my secret key and client id which use to setup google auth with passport

#### How should this be manually tested?
Tell us how other people can test out this your feature to see if it's working as expected.
- git clone
- run npm install and add some keys to .env
- go to your browser and run localhost:800/api/v1/auth/google

#### Screenshots
take pictures of the implementation of the project and paste them here
![Capture46](https://user-images.githubusercontent.com/38535951/82666279-7fcfaf00-9c2d-11ea-8d0c-4744cd07ddf7.PNG)


Please clear out all the descriptions, including this one as soon as you're done.
